collector/conn_pool_stats_hosts.go

```diff
@@ -0,0 +1,64 @@
+// TODO: rename file to connpool_hosts ?
```

Comment:
I think the original name is better than the one you are proposing in this comment, at least it's more consistent